src/core/wstringcol.cc

```diff
@@ -115,7 +116,11 @@ char* writable_string_col::buffer_impl<T>::strbuf_ptr() const {
 template <typename T>
 void writable_string_col::buffer_impl<T>::write(const char* ch, size_t len) {
   if (ch) {
-    if (sizeof(T) == 4) xassert(len <= Column::MAX_ARR32_SIZE);
+    if (sizeof(T) == 4) {
+      (void) len;
```

Comment:
Not sure what warning is this preventing? The variable `len` is clearly used on lines 124-126 below.

Reply:
I guess this one emitted a warning about an empty statement when you do `make build`, so I just used a noop on `len` to get rid of it.

Reply:
yeah, seems like ignoring it is a simpler alternative. We can even make it conditional on the build type: ignore in regular builds, leave as-is in debug builds

Reply:
I guess `(void) len;` was just less code :)

Reply:
I think I've seen multiple examples like this in this PR. Plus, consider this also from the perspective of trying to eliminate future such warnings as well.

Reply:
Yes, but now we have one additional line of code per such a case. With push/disable/pop pragmas it will be at lease seven lines per a case. Though we can define push/disable/pop as another macro :)